graph.js

```diff
@@ -137,21 +154,44 @@ app
     }
   })
 
+/**
+ * graph auth
+ */
+app
+  .command('auth [NODE] [AUTH_TOKEN]')
+  .description("Sets the access token to use when deploying to a Graph node")
+  .action((node, accessToken) => {
+    if (
+      accessToken === undefined ||
+      node === undefined ||
+      accessToken.length > 200
+    ) {
+      outputAuthConfig(node, accessToken)
+      console.error('--')
+      console.error('For more information run this command with --help')
+      process.exitCode = 1
+      return
+    }
+    keytar.setPassword('graphprotocol-auth', node, accessToken).then(() => {
+      console.log("Access token saved for Graph node:", node)
```

Comment:
Did you run `prettier` against this? I see a mix of `"` and `'` in this file. Single quotes are the way to go.